Assets/SEE/Net/Actions/AddNodeNetAction.cs

```diff
@@ -46,19 +46,10 @@ public AddNodeNetAction
              Vector3 scale)
             : base()
```

Comment:
In our other review, we already found definitions that also had this kind of formatting of the parameter list, where the opening parenthesis is on a new line. Do you know what causes this formatting?

Reply:
Actually, I prefer the opening parenthesis on a new line. A parenthesis  at the end of a line with no other token following looks ugly to me. Well, that may be a matter of taste, but we neither have the opening curly brackets at the end of the line where loops or conditions occur.